geometry_msgs/Vector3 current_position
geometry_msgs/Vector3 target_position
---
float64 relative_distance
geometry_msgs/Vector3 robot_location
geometry_msgs/Vector3 target_location
